### ISSUE TITLE
Read more link is missing relevant a11y context 

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#1727]
See: https://github.com/mozilla/addons-frontend/blob/4e7234fadbfc468375b9b7901842fd515529208f/src/ui/components/ShowMoreCard/index.js#L59

There a prefix of "Expand to" that's hidden but accessible, however this doesn't amount to enough of a context for an assistive tech user who might jump to that link. 

To be accessible this link needs to include a reference to what content is being expanded.

**Comment by TonnesM** [comment]
Just a thought: please note that this string may cause issues for localizations when the first part is not visible / omitted. My locale and German for instance use something similar to "more to read" for the second part, which doesn’t make much sense. I.o.w. using stand-alone "Read more" would require other wording, so splitting them may make some sense. See the current `de` or `fr` mobile versions for examples.

**Comment by AlexandraMoga** [comment]
I've observed that the comma before "Read more" link reproduces for other languages too. I thought this would be worth mentioning here.

Here are some examples:
https://addons-dev.allizom.org/dsb/firefox/addon/stylish/
https://addons-dev.allizom.org/hsb/firefox/addon/stylish/
https://addons-dev.allizom.org/hu/firefox/addon/stylish/

![image](https://user-images.githubusercontent.com/31961530/31126003-0209ef20-a853-11e7-9248-1e6d156f6bdf.png)

Note: Reproduces on all AMO servers, in FF56 (Win10 and Android 7.0)

**Comment by TonnesM** [comment]
The comma issue should be fixed in issue #2597. It just seems these 3 locales haven’t updated their strings properly (while others have):

https://pontoon.mozilla.org/dsb/amo-frontend/all-resources/?search=expand&string=162995
https://pontoon.mozilla.org/hsb/amo-frontend/all-resources/?search=expand&string=162995
https://pontoon.mozilla.org/hu/amo-frontend/all-resources/?search=expand&string=162995

I’ll email them to notify. Thanks!

**Comment by meskobalazs** [comment]
I fixed it in HU.
